docs/metrics/v3.md

```diff
@@ -20,378 +22,384 @@ Some metrics are bucket specific. These will have a `/bucket` component in their
+| `minio_cluster_usage_buckets_objects_count`                     | Total objects count in bucket. <br><br>Type: gauge                                    | `bucket`       |
+| `minio_cluster_usage_buckets_versions_count`                    | Total object versions (including delete markers) count in bucket. <br><br>Type: gauge | `bucket`       |
+| `minio_cluster_usage_buckets_delete_markers_count`              | Total delete markers count in bucket. <br><br>Type: gauge                             | `bucket`       |
+| `minio_cluster_usage_buckets_quota_total_bytes`                 | Total bucket quota in bytes. <br><br>Type: gauge                                      | `bucket`       |
+| `minio_cluster_usage_buckets_object_size_distribution`          | Bucket object size distribution. <br><br>Type: gauge                                  | `range`, `bucket` |
+| `minio_cluster_usage_buckets_object_version_count_distribution` | Bucket object version count distribution. <br><br>Type: gauge                         | `range`, `bucket` |
+
+#### `/cluster/usage/objects`
+
+| Name                                                     | Description                                                                         | Labels  |
+|----------------------------------------------------------|-------------------------------------------------------------------------------------|---------|
+| `minio_cluster_usage_objects_since_last_update_seconds`  | Time since last update of usage metrics in seconds. <br><br>Type: gauge             |         |
+| `minio_cluster_usage_objects_total_bytes`                | Total cluster usage in bytes. <br><br>Type: gauge                                   |         |
+| `minio_cluster_usage_objects_count`                      | Total cluster objects count. <br><br>Type: gauge                                    |         |
+| `minio_cluster_usage_objects_versions_count`             | Total cluster object versions (including delete markers) count. <br><br>Type: gauge |         |
+| `minio_cluster_usage_objects_delete_markers_count`       | Total cluster delete markers count. <br><br>Type: gauge                             |         |
+| `minio_cluster_usage_objects_buckets_count`              | Total cluster buckets count. <br><br>Type: gauge                                    |         |
+| `minio_cluster_usage_objects_size_distribution`          | Cluster object size distribution. <br><br>Type: gauge                               | `range` |
+| `minio_cluster_usage_objects_version_count_distribution` | Cluster object version count distribution. <br><br>Type: gauge                      | `range` |
 
-These are metrics about the minio logger webhooks
+### Debug metrics
 
-| Path              | Description                        |
-|-------------------|------------------------------------|
-| `/logger/webhook` | Metrics related to logger webhooks |
+Go runtime metrics for debugging.
+For details of individual metrics, see TODO
```

Comment:
`see TODO`?

Reply:
It would be good to have a link here, since we don't enumerate the individual metrics. Still trying to determine what.

I think they come from a Go method that dumps whatever runtime metrics it knows about at the time.